src/test/java/PlaylistTests.java

```diff
@@ -1,2 +1,196 @@
-public class PlaylistTests {
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PlaylistTests extends BaseTest{
+
+    @Test(enabled = true)
+    public void addSongTest() throws InterruptedException {
+        String playlistName = "Homework";
+
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        BasePage basePage = new BasePage(driver);
+        Actions acts = new Actions(driver);
+
+        //Login valid credentials
+        loginPage.login();
+        Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
+
+        //Click on "+" icon to create a playlist
+        basePage.clickPlusIcon();
+
+        //Click "New Playlist"
+        basePage.clickNewPlaylist();
+
+        //Input playlist name
+        WebElement nameField = driver.findElement(By.xpath("//input[@name='name']"));
+        nameField.clear();
+        nameField.sendKeys(playlistName, Keys.ENTER);
+        Thread.sleep(2000);
+
+        //Click "All Songs"
+        basePage.findElement(By.xpath("//li/a[text()='All Songs']")).click();
+
+        //Click "Reactor" song
+        WebElement reactorSong = driver.findElement(By.xpath("//*[@id='songsWrapper']/div/div/div[1]/table/tr[12]/td[2]"));
+        reactorSong.click();
+
+        //Right-click song
+        acts.contextClick(reactorSong).perform();
+
+        //Click "Add To"
+        basePage.findElement(By.xpath("//ul/li[@class='has-sub']")).click();
+
+        //Click "Homework"
+        basePage.findElement(By.xpath("//ul/li[@class='has-sub']/ul/li[7]")).click();
+
+        //Validate "Reactor" song is added
+        basePage.assertAddDeleteMessage();
```

Comment:
Please, use the POM pattern uses page objects that represent a web page (or a form within a page, if applicable) to separate test code (validations and test flow logic, for example) from page specific code. It does so by making all actions that can be performed on a page available as methods of the page object representing that page.

For example:



LoginPage = LoginTest

HomePage = HomeTest

CurrentQueuePage = CurrentQueueTest

![image](https://user-images.githubusercontent.com/104761482/214278344-60bfb74a-0d0d-40e4-9325-457a63b00f51.png)